GoodVibes.Client.Wpf/applicationSettings.json

```diff
@@ -6,9 +6,9 @@
     "SyncHubPath": "/syncHub",
     "PiShockHubPath": "/piShockCommandHub"
   },
-  "OscSettings": {
-    "ServerPort": 9001,
-    "ClientPort": 9000
+  "LovenseSettings": {
+    // I wouldn't recommend going to low(250 for 4 calls a second).

```

Comment:
😅 Comments are a bit of a risky business in a JSON File, if the language and VS is fine with it, I'd keep it.
But a lot of editors might be unhappy seeing this.

I totally agree with why you included it.

Reply:
IDE's definitely don't like it, Visual Studio included :) But it runs. But the language is completely fine with it :D